frontend/src/api/mockAPI/utils.js

```diff
@@ -245,6 +246,19 @@ export class MockAPIController {
     validateNew() {
         throw new Error("Subclasses must impliment `validateNew()`");
     }
+
+    /**
+     * Validates a property of an item instance. Throws an error
+     * if this property are invalid/incompatable.
+     *
+     * @param {object} obj
+     * @param {string} prop
+     * @memberof MockAPIController
+     */
+    // eslint-disable-next-line no-unused-vars
+    validateProp(obj, prop) {
```

Comment:
Why are you passing in an object? I don't get it...

Reply:
say we want to update a session. `obj` here is the request body `{ id: xxx, name: xxx}`. `prop` is the key/field. I need to get the value of the field.

Reply:
You should be passing in the prop and the value, not the prop and the object.

Reply:
right... That sounds way better